hr_attendance_report_theoretical_time/wizards/wizard_theoretical_time.py

```diff
@@ -49,7 +54,9 @@ def view_report(self):
         action = self.env.ref(
             "hr_attendance_report_theoretical_time." "hr_attendance_theoretical_action"
         ).read()[0]
-        action["domain"] = [("employee_id", "in", self.employee_ids.ids)]
+        action["domain"] = [
+            ("employee_id", "in", self.with_context(active_test=False).employee_ids.ids)
```

Comment:
I don't think we should add here the active_test context, as both behaviors will be accepted and rejected depending on the user base, and this a change of behavior.

Reply:
Sorry, my bad. I understood other thing.